set-postgres-permissions.bash

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+export AZURE_CONFIG_DIR=~/.azure-db-manager
+az login --identity
+
+
+# shellcheck disable=SC2155
+export PGPASSWORD=$(az account get-access-token --resource-type oss-rdbms --query accessToken -o tsv)
+
+SQL_COMMAND="
+DO
+\$do\$
+BEGIN
+   IF NOT EXISTS (
+      SELECT FROM pg_catalog.pg_roles  -- SELECT list can be empty for this
+      WHERE rolname = '${DB_READER_USER}') THEN
+
+      select * from pgaadauth_create_principal('${DB_READER_USER}', false, false);
+      
+   END IF;
+END
+\$do\$;
+
+ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO PUBLIC;
+REVOKE CREATE ON SCHEMA public FROM public;
```

Comment:
you still need that grant as well I think for existing tables
https://github.com/hmcts/cnp-module-postgres/blob/40306e4774dbde9a5c164cb7bd3b513e4da5cbf8/set-postgres-permissions.bash#L23

Reply:
updated script for this